### DISCUSSION TITLE
New Website

### DISCUSSION THREAD

**Discussion by ad3154** [Open-Agriculture/AgIsoStack-plus-plus#85] [Announcements]
Check out our new website! https://isobusplusplus.com

This is part of a broader effort to apply branding to the project that reflects its position in the industry as the only free C++ ISOBUS CAN stack (that I know of).

**Comment by tomstix** [comment]
Do you know IsoAgLib? It's (with exceptions) GPL-Licensed and maintaind by OSB-AG who are also commercially developing ISOBUS applications. I haven't dug into it too deep but it might help the development of this library. This is the svn repository: [IsoAgLib](https://projects.osb-ag.de/svn/OSB/IsoAgLib/IsoAgLib/)

**Reply by ad3154** [reply]
Actually, I have heard the name once before, on this project: https://github.com/moehman/PoolEditSrc But I have never really looked into it. I will check it out! Thanks! We do have to be a bit careful about using it as a resource if it's GPL, as our license is more permissive than theirs.
